crmsh/sbd.py

```diff
@@ -184,7 +200,7 @@ def _determine_sbd_watchdog_timeout(self):
         if utils.is_qdevice_configured() and utils.service_is_active("corosync-qdevice.service"):
             qdevice_sync_timeout = utils.get_qdevice_sync_timeout()
             self._sbd_watchdog_timeout = qdevice_sync_timeout + 5
-            if self._is_s390 and self._sbd_watchdog_timeout < 15:
+            if self._context.is_s390 and self._sbd_watchdog_timeout < 15:
                 self._sbd_watchdog_timeout = 15
             self._stonith_timeout = self.calculate_stonith_timeout(self._sbd_watchdog_timeout)
         # add sbd and qdevice together from beginning
```

Comment:
The main point of the current _determine_sbd_watchdog_timeout() is trying to update _sbd_watchdog_timeout based on qdevice_sync_timeout. It worth to send a message to user sbd_watchdog_timeout is changed because of this.

I think this is also valid to disk-based-sbd too. Loop in @gao-yan to double check, TIA!

Reply:
> The main point of the current _determine_sbd_watchdog_timeout() is trying to update _sbd_watchdog_timeout based on qdevice_sync_timeout. It worth to send a message to user sbd_watchdog_timeout is changed because of this.

Makes sense.

> 
> I think this is also valid to disk-based-sbd too. Loop in @gao-yan to double check, TIA!

I don't think it's needed for sbd with disk.

For the specific situation:
https://github.com/corosync/corosync-qdevice/issues/21#issuecomment-842282435

, in case of diskless sbd, it purely relies on update/loss of quorum to get itself reset to achieve fencing. But for the case sbd with disk, because of the different mechanism, the failing node will eat the poison pill and already get fenced before sync timeout is popped...

Reply:
Thanks Yan, I have much better understanding of the point here now. Though, I think I will need more time to understand the complete picture in related to qdevice-sync-timeout. 

It looks things get simplified quite a bit in sbd-1.5.0 + corosync-3, the part with qdevice-sync-timeout will be deleted  then, though I'm 100% sure. If so, could be better to add a inline TODO to revisit this in sbd-1-5.0 + corosync-3 context.